iot/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.enmasse</groupId>
+    <artifactId>enmasse</artifactId>
+    <version>0.27-SNAPSHOT</version>
+  </parent>
+  <artifactId>iot</artifactId>
+  <packaging>pom</packaging>
+
+  <properties>
+    <hono.version>0.8</hono.version>
+    <logback.version>1.2.3</logback.version>
+    <spring.boot.version>1.5.18.RELEASE</spring.boot.version>
+    <spring.version>4.3.19.RELEASE</spring.version>
+    <micrometer.version>1.0.6</micrometer.version>
+  </properties>
+
+  <modules>
+    <module>tenant-service</module>
+  </modules>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.eclipse.hono</groupId>
+        <artifactId>hono-service-base</artifactId>
+        <version>${hono.version}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-logging</artifactId>
+        <version>${spring.boot.version}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter</artifactId>
+        <version>${spring.boot.version}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>io.micrometer</groupId>
+        <artifactId>micrometer-registry-prometheus</artifactId>
+        <version>${micrometer.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.springframework</groupId>
+        <artifactId>spring-web</artifactId>
+        <version>${spring.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.springframework</groupId>
+        <artifactId>spring-webmvc</artifactId>
+        <version>${spring.version}</version>
+      </dependency>
+
```

Comment:
Can we please get rid of all of the above dependencies? No spring boot or micrometer please :) We've managed without it so far, and we just got rid of spring boot from mqtt-gateway and mqtt-lwt.

Reply:
Hono uses Spring Boot and Micrometer extensively. So I don't see much benefit in dropping and rewriting all the Hono base classes that we currently use. If we use Hono, then Spring Boot and Micrometer are a part of that.

Reply:
As discussed, we'll have to live with it ... for now :)